### PR TITLE
Proposal: deprecate and remove `feature-exists`

### PR DESCRIPTION
**Pull request by stof**
https://github.com/sass/sass/issues/3702

### CODE REVIEW

**Comment by stof** [comment]
@nex3 even though the process says that any proposal should have a `Syntax` or `Semantics` section, I don't see what I should put in the `Semantics` section when the only change is removing a function. Is the `Deprecation Process` section enough ?

File: proposal/remove-feature-exists.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Background
+
+> This section is non-normative.
+
+`feature-exists` is ill-defined as nothing specifies what each feature actually
+requires to be considered as supported. The shared `sass-spec` testsuite does
+not cover this either. Thus, no new feature identifier has been added in it
+since years.
+New Sass features essentially fall into one of three categories:
```
Comment (nex3) [inline proposal/remove-feature-exists.md:23]: Style nit: empty line between paragraphs.

File: proposal/remove-feature-exists.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  * [`feature-exists`](#feature-exists)
+
+## Background
+
+> This section is non-normative.
+
+`feature-exists` is ill-defined as nothing specifies what each feature actually
+requires to be considered as supported. The shared `sass-spec` testsuite does
+not cover this either. Thus, no new feature identifier has been added in it
+since years.
```
Comment (nex3) [inline proposal/remove-feature-exists.md]: Nit: "for years"

**Review by nex3 — APPROVED** [review]

File: proposal/remove-feature-exists.md (hunk 2 above)
Comment (stof) [reply proposal/remove-feature-exists.md]: I fixed those feedbacks

**Comment by stof** [comment]
@nex3 this proposal is ready.

**Review by nex3 — APPROVED** [review]

**Pull request merged**